features/dd-sdk-android-trace-api/src/main/kotlin/com/datadog/android/trace/api/scope/DataScopeListener.java

```diff
@@ -9,9 +9,7 @@
 /** Hooks for scope activation */
 public interface DataScopeListener {
   /**
-   * Called just after a scope becomes the active scope
-   * May be called multiple times. When a scope is initially created, or after a child scope is
-   * deactivated.
```

Comment:
it cannot be called multiple times anymore?

Reply:
Looks like lost this changes in one of updates, restored